ponyo/train_vae_modules.py

```diff
@@ -26,6 +29,33 @@ def fxn():
     fxn()
 
 
+def set_all_seeds(np_seed=42, rn_seed=12345, tf_seed=1234):
```

Comment:
Is there a reason that each of the default seeds are different? 

Also, is there a situation where you would want the seeds to be set independently instead of just passing in a single number and saying "set all seeds to this"?

Reply:
I believe these were legacy values I used, so I don't think there is a logic, as far as I know, to set one seed independently from another if the goal is to generate a reproducible model. Thank you for catching this